cmd/run.go

```diff
@@ -274,11 +281,34 @@ func pollTask(c *client.Client, id string, timeout time.Duration, showlogs, remo
 
 		fmt.Println("ID: ", item.ID, " Update at: ", item.UpdatedAt)
 		for {
+			if len(taskRequest.HeartbeatUrl) > 0 && len(taskRequest.JobToken) > 0 {
+				heartbeatClient := &http.Client{}
```

Comment:
the client should be reused across requests, usually. Also you might want to specify read/write timeouts to avoid waiting too long.